CONTRIBUTING.md

```diff
@@ -23,71 +23,70 @@ We welcome participation in an open project. We want to make it as easy as possi
 
 cd ~ # get to your home directory or where ever you want to go
 
-git clone https://github.com/YOURACCOUNT/covid-safe-paths
+git clone git@github.com:YOURACCOUNT/covid-safe-paths.git
 
 # change into the newly created directory
 cd covid-safe-paths
 
 # set upstream against COVID Safe Paths repository
-git remote add upstream https://github.com/tripleblindmarket/covid-safe-paths.git
+git remote add upstream git@github.com:tripleblindmarket/covid-safe-paths.git
 
 ```
 
 (see also how to [fork a repo](https://help.github.com/articles/fork-a-repo/))
 
 ## Make Changes
 
-1. Create a branch based on the `develop` branch on your forked repository. Name the branch something to reflect what you are doing. For example, if you want to add a new icon, a branch name you could use:
+### Create a branch 
+
+1. Always create a new branch from the latest `upstream/develop`:
+  ```bash
+  git checkout develop # you want to branch from the latest 'develop' branch
+  
+  git pull upstream/develop # make sure you have the latest code from upstream
+  
+  git push origin develop # optional, push these changes to YOUR fork's develop branch
+  ```
+2. Create the branch. Name the branch something to reflect what you are doing.
+  ```
+  git checkout -b "feature/new-icon" develop # new branch created!
+  
+  "or"
+  
+  git checkout -b "fix/new-icon" develop # new branch created!
+
+  "or"
+
+  git checkout -b "release/new-icon" develop # new branch created!
+  ```
+3. Stick to the coding style and patterns that are used already.
+4. Document code! Comments are good. More comments are better. :)
+5. Make commits as you desire. Ultimately they will be squashed, so make notes to yourself. It's as simple as `git commit -m "commit message goes here"`!
+
+### Merge upstream/develop into your branch to get the latest changes.
 
 ```bash
-git checkout develop # you want to branch from the main 'develop' branch
+# if you've already done this it will fail, that's fine:
+git remote add upstream git@github.com:tripleblindmarket/covid-safe-paths.git
 
-git pull # make sure you have the latest code when you start the branch
+git checkout feature/my-feature
```

Comment:
Not sure this is always necessary... good to be safe.  Maybe this comment above this line ... 
`# ensure you are on the correct branch`

Reply:
I wanted to be extra sure that they did not do this on develop, will add the comment